lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -210,28 +205,30 @@ class CategoryRenderer {
    * Renders the group container for a group of audits. Individual audit elements can be added
    * directly to the returned element.
    * @param {!ReportRenderer.GroupJSON} group
+   * @param {{expandable: boolean}} opts
    * @return {!HTMLDetailsElement}
    */
-  _renderAuditGroup(group) {
-    const auditGroupElem = /** @type {!HTMLDetailsElement} */ (this._dom.createElement('details',
+  _renderAuditGroup(group, opts) {
+    const expandable = opts.expandable;
+    const auditGroupElem = /** @type {!HTMLDetailsElement} */ (
+          this._dom.createElement(expandable ? 'details' :'div',
           'lh-audit-group lh-expandable-details'));
```

Comment:
Since this is shared by expandable and non-expandable, `lh-expandable-details` -> `lh-group` (or something better)? Is `lh-audit-group` only used for these?